Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    
+    name: "PerimeterX",
```

Comment:
Is this the same package name as in cocopods? if so, good!

Reply:
@AsafAklerPX, yes.